### PR TITLE
Add auto close timer on Feedback thanksModal

### LINKED ISSUE #1157: The FEEDBACK-END screen does not disappear after 3 seconds [DESKTOP] [ MOBILE]
**Environment:**  https://staging.fightpandemics.work/

**Expected  Results:**  System visualizes **FEEDBACK-END** screen which disappears after 3 seconds.

**Actual Results:** The FEEDBACK-END screen does not disappear after 3 seconds 

**Screenshots :** 

**MOBILE**
image

**DESKTOP**
image

**EXPECTED**
image

### PR DESCRIPTION
**Pull request by jyclam**
### What does this pull request aim to achieve? 💯
Feedback thanksModal closes after 3 seconds 

_Please be concise and link any related issue(s):_
closes #1157

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/templates/layouts/NavigationLayout.js [hunk 1]
```diff
@@ -622,12 +623,12 @@ const NavigationLayout = (props) => {
...
-  if (isAuthenticated) { 
-    tagManagerArgs['dataLayer'] = {userId: user.id};
+  if (isAuthenticated) {
+    tagManagerArgs["dataLayer"] = { userId: user.id };
     TagManager.initialize(tagManagerArgs);
-  } 
+  }
 
-  return <>{renderNavigationBar()}</>; 
+  return <>{renderNavigationBar()}</>;
```
Comment (idjevm) [inline client/src/templates/layouts/NavigationLayout.js:631]: This is most likely going to have merge conflicts / looks like you might've started this branch off master instead of rc/v1.1. But yeah please undo the changes besides Line 306.

Comment (jyclam) [reply client/src/templates/layouts/NavigationLayout.js:631]: No, this was branched from rc/v1.1.

```
44b8731e (HEAD -> feature/fix-post-alignment-and-background, origin/feature/fix-post-alignment-and-background) Tweak PostCard padding
22ee0ebc Change PostCard margin to padding, remove post title side margin & padding
409b6c97 (origin/rc/v1.1, rc/v1.1) add scroll to auto in create post (#1216)
```

Comment (idjevm) [inline client/src/templates/layouts/NavigationLayout.js:631]: I still think that this will cause some sort of conflict once we try to merge either this to master or master to the rc branch. This code will no longer exist, and hence it will cause conflicts... Can you please just revert the formatting changes from lines 625 to 631, and just have the line 306?

Comment (jyclam) [reply client/src/templates/layouts/NavigationLayout.js:631]: Sure, I think I can edit directly on Github and avoid our auto linting.

Comment (jyclam) [reply client/src/templates/layouts/NavigationLayout.js]: Done!

**Comment by idjevm** [comment]
Thanks John!
